library/public/engine.h

```diff
@@ -22,16 +22,26 @@ namespace f3d
  * Configured on creation using an enum, then all objects
  * can be accessed through their getter.
  *
- * Example usage:
+ * Example usage for a default scene file:
  *
  * \code{.cpp}
  *  f3d::engine eng(f3d::engine::CREATE_WINDOW | f3d::engine::CREATE_INTERACTOR);
  *  f3d::loader& load = eng.getLoader();
- *  load.addFile("path/to/file");
- *  load.loadFile(f3d::loader::LoadFileEnum::LOAD_CURRENT);
+ *  load.resetToDefaultScene().addGeometry("path/to/file");
```

Comment:
`resetToDefaultScene()` is needed even if nothing is added yet?

Reply:
yes, see the addGeometry comment.